sql/core/benchmarks/FunctionBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,60 @@
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+Java scalar function (long + long) -> long result_nullable = true codegen = true:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+----------------------------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                                     52937          54250        1181          9.4         105.9       1.0X
+with long_add_magic                                                                       20527          20975         683         24.4          41.1       2.6X
+with long_add_static_magic                                                                20513          20944         447         24.4          41.0       2.6X
+
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+Java scalar function (long + long) -> long result_nullable = false codegen = true:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                                      54502          54710         208          9.2         109.0       1.0X
+with long_add_magic                                                                        21790          22062         414         22.9          43.6       2.5X
+with long_add_static_magic                                                                 19014          19715         626         26.3          38.0       2.9X
+
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+Java scalar function (long + long) -> long result_nullable = true codegen = false:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                                      64039          64306         293          7.8         128.1       1.0X
+with long_add_magic                                                                       199121         199232         144          2.5         398.2       0.3X
```

Comment:
A few optimizations can be done to bring this to ~0.8. I'll do it as a follow-up.

Reply:
If that is orthogonal to this PR, let's have a separate PR for that instead of the follow-up.

Reply:
As you know, @sunchao . This is a request as one of the downstream users. I hope each patch is orthogonally backportable.

Reply:
Yeah by "follow-up" I mean a separate PR :)